app/uk/gov/hmrc/leakdetection/controllers/WebhookController.scala

```diff
@@ -55,8 +55,11 @@ class WebhookController @Inject()(
           )
 
         case pushUpdate: PushUpdate =>
-          scanningService.queueRequest(pushUpdate).map { _ =>
-            Ok(toJson(WebhookResponse("Request successfully queued")))
+          scanningService.queueDistinctRequest(pushUpdate).map { duplicate =>
+            if (duplicate)
+              Ok(toJson(WebhookResponse("Duplicate request ignored")))
```

Comment:
can we have a log of this case - I think it would be good to see if this is really happening